src/components/NeedsGameWorld.cpp

```diff
@@ -0,0 +1,41 @@
+#include "NeedsGameWorld.hpp"
+#include <exception/Throw.hpp>
+#include <RTTI/RTTI_NeedsGameWorld.hpp>
+#include <components/GameWorld.hpp>
+
+namespace REGoth
+{
+  NeedsGameWorld::NeedsGameWorld(const bs::HSceneObject& parent)
+      : bs::Component(parent)
+  {
+    setName("NeedsGameWorld");
+    findGameWorld();
```

Comment:
Maybe move the logic to `onInitialized()` (although it shouldn't make a big difference)